app/assets/javascripts/countries/views/CountryPantropicalView.js

```diff
@@ -24,7 +25,16 @@ define([
       $('#view_selection').find('.btn').removeClass('active');
       $(e.target).addClass('active');
       $('#vis').find('.' + $(e.target).attr('id')).show();
-      toggle_view($(e.target).attr('id'));
+
+      var viewId = $(e.target).attr('id');
+      toggle_view(viewId);
+
+      if(viewId === 'change') {
+        $('#vis').addClass(viewId);
+        var totalEmissionsChart = new PantropicalTotalEmissionsView();
```

Comment:
are we creating the graph every single time we click the CHANGE tab? I'd rather throw it once and then toggle it if we want to show or hide it.